public/app/plugins/datasource/cloud-monitoring/components/Experimental/GroupBy.tsx

```diff
@@ -38,7 +38,6 @@ export const GroupBy: FunctionComponent<Props> = ({
         <EditorField
           label="Group by"
           tooltip="You can reduce the amount of data returned for a metric by combining different time series. To combine multiple time series, you can specify a grouping and a function. Grouping is done on the basis of labels. The grouping function is used to combine the time series in the group into a single time series."
-          htmlFor={`${refId}-group-by`}
```

Comment:
why did you remove this `htmlFor`?

Reply:
The `EditorField` automatically gets the child input id. The `getByLabelText` in the tests would fail if there was no input element associated with the label.